docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
+current status of the action. Additionally, functions returned from
+register will be used as the handler for a Microcosm dispatch:
+
+```javascript
+import { addPlanet } from '../actions/planets'
+
+const StatusLog = {
+  //...
+  register (command, status) {
```

Comment:
I think it'd be fine to make this just for Effects, until we discover a pressing need to extend it to Domains. You can always push an ~~action~~ command from an effect to update state. It definitely makes more sense in my head for Effects.